server/views/admin-add-category.hbs

```diff
@@ -1,21 +1,21 @@
 <form method="post">
         <div class="form-group has-success">
 
-        <h5 class="text-success" >Title:</h5>
-        <textarea  class="form-control form-control-success" id="inputSuccess1" name="title"  cols="2" rows="1">
-    </textarea>
-    <br> 
+        <h5 class="text-success  mt-5" >Title:</h5>
+         <input class="col-lg-4 col-md-8 mb-4" name="title" value={{this.article.title}}>
+
+   
+    
          <h5 class="text-success" >Short Description:</h5>
-     <br>
-       <textarea  class="form-control form-control-success" id="inputSuccess1" name="shortDescription" >
-    </textarea>
-    <br><br>
+   
```

Comment:
then also instead of having `h5` for title - just use form label

```
<form>
  <div class="form-group">
    <label class="col-form-label" for="formGroupExampleInput">Example label</label>
    <input type="text" class="form-control" id="formGroupExampleInput" placeholder="Example input">
  </div>
  <div class="form-group">
    <label class="col-form-label" for="formGroupExampleInput2">Another label</label>
    <input type="text" class="form-control" id="formGroupExampleInput2" placeholder="Another input">
  </div>
</form>
```